groovy/material/UnknownCompositionMaterials.groovy

```diff
@@ -1085,7 +1085,7 @@ public class UnknownCompositionMaterials {
                 .color(0x696840)
                 .build();
 
-        UraniumVanadiumExtractionMixture = new Material.Builder(4210, SuSyUtility.susyId('uranium_vanadium_extraction_mixture'))
+        UraniumExtractionMixture = new Material.Builder(4210, SuSyUtility.susyId('uranium_extraction_mixture'))
```

Comment:
should we make a data fixer to convert existing uranium_vanadium_extraction_mixture to uranium_extraction_mixture

Reply:
also do we need to rename this mixture, considering it is only being used to process carnotite?

Reply:
bruh it is used for both, also only like the fluid will just mutate into the new one since it shares the ID

Reply:
only GT items share the ID, fluids are registered via fluid names directly.

Reply:
ok but like 2 ppl on the team know how to write a datafixer and they're both busy so meh

Reply:
^

Reply:
fair point then.
should we remove the ORE flag of uraninite ore too?

Reply:
might be prudent to remove uraninite entirely since pitchblende is a uraninite ore

Reply:
need to add a disable decomp flag to pitchblende too